Create README.md

tarikuzuma opened the pull request:
Created installation guidelines

tarikuzuma committed:
Create README.md

Created installation guidelines

fofajardo commented on README.md:
```diff
@@ -0,0 +1,37 @@
+# README
```
nit: use full game name here instead of just readme.

fofajardo reviewed (changes requested):
Please see review comments. Otherwise, LGTM!

tarikuzuma committed:
Update and rename README.md to Dikta.md

Also changed "- If you encounter any specific errors, try searching online for solutions or consult the game's documentation (if available)."

tarikuzuma commented:
I changed readme.md to Dikta.md, can you check the commit?

tarikuzuma committed:
Update and rename Dikta.md to README.md

tarikuzuma committed:
Update README.md

tarikuzuma committed:
Update README.md

Updated heading

fofajardo reviewed (approved):
LGTM! Merging.
